src/components/general/spritesheet/Spritesheet.jsx

```diff
@@ -11,8 +12,10 @@ export const Spritesheet = ({
     enabled,
     size,
     numFrames,
+    animated = true,
```

Comment:
IMO there shouldn't be a non-animated spritesheet. Something that is not animated should use something other than a spritesheet.

Reply:
yup, removed.